variables.tf

```diff
@@ -249,4 +493,90 @@ variable "integration_subnet_id" {
   type        = string
   default     = null
   description = "The resource ID of the subnet"
+}
+
+variable "app_insights_workspace_id" {
```

Comment:
Where is description?
Please add in all and also please follow the following format:-
```
type        = ""
default     = ""
description = ""
```